DepotDownloader/ContentDownloader.cs

```diff
@@ -1258,7 +1258,7 @@ private static async Task DownloadSteam3AsyncDepotFileChunk(
                             break;
                         }
 
-                        Console.WriteLine("Encountered error downloading chunk {0}: {1}", chunkID, e.StatusCode);
+                        Console.WriteLine("Encountered {1} for chunk {0}. Retrying.", chunkID, e.StatusCode);
```

Comment:
```suggestion
                        Console.WriteLine("Encountered HTTP {1:D} for chunk {0}. Retrying...", chunkID, e.StatusCode);
```

Reply:
This is definitely not gonna be work out, because StatusCode is a value like `ServiceUnavailable` or `NotFound` from what I've observed.